components/brave_rewards/browser/rewards_service_impl.h

```diff
@@ -72,9 +72,10 @@ using GetDebugCallback = base::Callback<void(bool)>;
 using GetReconcileTimeCallback = base::Callback<void(int32_t)>;
 using GetShortRetriesCallback = base::Callback<void(bool)>;
 using GetTestResponseCallback =
-    base::Callback<void(const std::string&,
-                        std::string*,
-                        std::map<std::string, std::string>*)>;
+    base::Callback<void(const std::string& url,
+                        int* code,
```

Comment:
nit: `response_status_code` instead of `code`

Reply:
done